integration_tests/__init__.py

```diff
@@ -78,8 +78,10 @@ def get_output_ignoring_non_zero_exit(self, binary, cwd):
             output = exception.output
         return output
 
-    def login(self, email='u1test+snapcraft@canonical.com',
+    def login(self, email=None,
               password=None, expect_success=True):
+        email = email or os.getenv('TEST_USER_EMAIL',
+                                   'u1test+snapcraft@canonical.com')
```

Comment:
Ha right.

This is a first step to explicit the assumptions made in the test suite against the staging server *content*.
Things like:
- those tests can't be run against production,
- the user must have registered the 'aseemble' and 'basic' names,
- the user must have register the 'femto' name and have published one upload
- may be others
Which are not currently documented.